x-pack/plugins/security_solution/cypress/README.md

```diff
@@ -208,6 +242,44 @@ Because of `cy.exec`, used to invoke `es_archiver`, it's necessary to override i
 
 > Warning: Setting the NODE_TLS_REJECT_UNAUTHORIZED environment variable to '0' makes TLS connections and HTTPS requests insecure by disabling certificate verification.
 
+### CCS
+
+Tests running in CCS configuration need to care about two aspects:
+
+1. data (eg. to trigger alerts) is generated/loaded on the remote cluster
+2. queries (eg. detection rules) refer to remote indices
+
+Incorrect handling of the above points might result in false positives, in that the remote cluster is not involved but the test passes anyway.
+
+#### Remote data loading
+
+Helpers `esArchiverCCSLoad` and `esArchiverCCSUnload` are provided by [cypress/tasks/es_archiver.ts](https://github.com/elastic/kibana/blob/master/x-pack/plugins/security_solution/cypress/tasks/es_archiver.ts):
+
+```javascript
+import { esArchiverCCSLoad, esArchiverCCSUnload } from '../../tasks/es_archiver';
+```
+
+They will use the `CYPRESS_CCS_*_URL` environment variables for accessing the remote cluster. Complex tests involving local and remote data can interleave them with `esArchiverLoad` and `esArchiverUnload` as needed.
```

Comment:
Is my understanding correct that `esArchiverCCSLoad, esArchiverCCSUnload` can be used to work with any es archive (including already existing archives)? The only difference that they work with the remote cluster instead of the local one.

Reply:
@banderror that is correct, you can use any archive and the data is going to be loaded on the remote.